dashboard/test/factories/user_ml_models.rb

```diff
@@ -0,0 +1,18 @@
+# == Schema Information
```

Comment:
We have very few files in this directory.  It might be worth holding off adding this file until we know we will use it.